### ISSUE TITLE
Test against x32 ABI on Travis

### ISSUE THREAD

**Issue by omasanori** [picrin-scheme/picrin#286]
After https://github.com/travis-ci/travis-ci/issues/2046 is fixed, we will gain newer compilers, libraries and kernel on Travis, which means that we can test picrin against x32 ABI, a ILP32 processor-specific ABI for x86_64. We will get more chances to catch portability issues with it.

**Comment by wasabiz** [comment]
@omasanori 

Beautiful. How can we add such architectures to the testing list? .travis.yml?

**Comment by omasanori** [comment]
Yes, just adding ``- CFLAGS="-mx32"`` to ``env`` in ``.travis.yml``.

``gcc-multilib`` has dependencies to ``libgcc`` and ``libc`` for x32 if supported, so no other changes will be required.
